protoc-gen-twirp/generator.go

```diff
@@ -352,6 +352,10 @@ func (t *twirp) generateUtils() {
 	t.P(`	// ProtocGenTwirpVersion is the semantic version string of the version of`)
 	t.P(`	// twirp used to generate this file.`)
 	t.P(`	ProtocGenTwirpVersion() string`)
+	t.P(`// PathPrefix is used for all URL paths on the generated server.`)
```

Comment:
I'm picky about comments. Could you phrase this as follows:

```go
// PathPrefix returns the HTTP URL path prefix for all methods handled by this
// service. This can be used with an HTTP mux to route twirp requests
// alongside non-twirp requests on one HTTP listener.
```

I realize that you've copied the comment that's on the `const`, but I think that comment could be improved too.

Reply:
I don't mind changing this.  Should I change the comment on the `const` as well?